dont include user auth info on cnpm

```
PS D:\> cnpm install express
npm http GET http://registry.cnpmjs.org/express
npm http 401 http://registry.cnpmjs.org/express
npm http GET http://registry.cnpmjs.org/express
npm http 401 http://registry.cnpmjs.org/express
npm ERR! Error: unauthorized Name or password is incorrect.: express
npm ERR! at RegClient.<anonymous> (C:\Program Files
(x86)\nodejs\node_modules\npm\node_modules\npm-registry-client\l
ib\request.js:272:14)
npm ERR! at Request.self.callback (C:\Program Files
(x86)\nodejs\node_modules\npm\node_modules\request\request.js:12
9:22)
npm ERR! at Request.EventEmitter.emit (events.js:98:17)
npm ERR! at Request.<anonymous> (C:\Program Files
(x86)\nodejs\node_modules\npm\node_modules\request\request.js:873:
14)
npm ERR! at Request.EventEmitter.emit (events.js:117:20)
npm ERR! at IncomingMessage.<anonymous> (C:\Program Files
(x86)\nodejs\node_modules\npm\node_modules\request\request
.js:824:12)
npm ERR! at IncomingMessage.EventEmitter.emit (events.js:117:20)
npm ERR! at _stream_readable.js:920:16
npm ERR! at process._tickCallback (node.js:415:13)
npm ERR! If you need help, you may report this log at:
npm ERR! <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR! <npm-@googlegroups.com>

npm ERR! System Windows_NT 6.1.7601
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe"
"C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\
\npm-cli.js" "--cache=C:\\Users\\meteor\\.npm\\.cache\\cnpm"
"--registry=http://registry.cnpmjs.org" "install" "express"

npm ERR! cwd D:\
npm ERR! node -v v0.10.22
npm ERR! npm -v 1.3.14
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR! D:\npm-debug.log
npm ERR! not ok code 0
PS D:\>
```

--- a/bin/cnpm-sync
+++ b/bin/cnpm-sync
@@ -66,6 +66,7 @@ npm.load({
   registry: registry,
   cache: config.cache,
   loglevel: 'warn',
+  userconfig: config.userconfig,
 }, function (err) {
   npm.registry.request('PUT', name + '/sync', {}, function (err, result) {
     if (err) {

--- a/config.js
+++ b/config.js
@@ -28,19 +28,5 @@ var config = module.exports = {
   cnpmHost: 'http://cnpmjs.org',
   cnpmRegistry: 'http://registry.cnpmjs.org',
   cache: path.join(root, '.npm/.cache/cnpm'),  //cache folder name
-  customConfig: path.join(root, '.cnpmrc.json')
+  userconfig: path.join(root, '.cnpmrc')
 };
-
-if (fs.existsSync(config.customConfig)) {
-  var custom;
-  try {
-    custom = require(config.customConfig);
-  } catch (e) {
-    custom = null;
-  }
-  if (custom) {
-    for (var k in custom) {
-      config[k] = custom[k];
-    }
-  }
-}

--- a/origin_npm.js
+++ b/origin_npm.js
@@ -28,6 +28,8 @@ var CWD = process.cwd();
 
 args.unshift('--registry=' + program.registry);
 args.unshift('--cache=' + config.cache);
+args.unshift('--userconfig=' + config.userconfig);
+
 var cmd = 'npm';
 if (process.platform === "win32") {
   cmd = 'npm.cmd';

